src/include/sof/platform.h

```diff
@@ -47,6 +47,24 @@ int platform_init(struct sof *sof);
 
 /** @}*/
 
+#define HOST_TO_LOCAL 0
```

Comment:
This is i.MX specific, even more is 8ULP specific.
Can we add this in a file imx specific?

Reply:
It should be common code, if we add new platform with need to convert address then this part code will be used. If we add it 
in 8ulp specific, and then we need make it be common when add new platform.

Reply:
For now is only for 8ulp, so add it just for 8ulp, let's not change the core code if is not needed there.
IF, at some point, will be needed for something else we can make it common, but now this has nothing to do with other imx targets or Intel's.

Reply:
Sure.